client/Containers/Cart/Summary.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import FeaturedButton from '../../components/styles/FeaturedButton';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { resetCart } from '../../store/cart/cart';
+// need a checkout thunk that converts carts to orders
+
+const mapStateToProps = (state) => {
+  return {
+    cart: state.cart,
+  };
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    resetCart: () => dispatch(resetCart()),
+  };
+};
+
+const Summary = ({ cart, resetCart }) => {
+  let { cartProducts } = cart;
+  cartProducts = cartProducts || [];
+
+  const orderTotal = () => {
+    if (!cartProducts.length) return 0;
+    return cartProducts.reduce(
+      (accum, item) => accum + item.quantity * item.product.price,
+      0
+    );
```

Comment:
Like above, modified this reduce function to work for all array lengths by starting with a default value and adding to it on each pass through the array